resources/references/app-reference/script-reference/product-script-services-reference.md

```diff
@@ -1,51 +1,242 @@
+
+    Returns the newly created `PriceCollection`.
+* **Arguments:**
+    * *`array`* **price**: The prices for the new collection, indexed by the currency-id or iso-code of the currency.
+* **Examples:**
+    * Create a new Price in the default currency.
+
+        {% raw %}
+        ```twig
+        {% set price = services.cart.price.create({
+		    'default': { 'gross': 19.99, 'net': 19.99}
+		}) %}
+        ```
+        {% endraw %}
+_________
 ## [`Shopware\Core\Content\Product\Hook\Pricing\PriceCollectionFacade`](https://github.com/shopware/shopware/blob/trunk/src/Core/Content/Product/Hook/Pricing/PriceCollectionFacade.php) {#pricecollectionfacade}
 
 The PriceCollectionFacade is a wrapper around the calculated price collection of a product. It allows to manipulate the quantity
 prices by resetting or changing the price collection.
 
+
 ### reset()
 
 * The `reset()` functions allows to reset the complete price collection.
 
+    
 ### change()
 
 * The `change()` function allows a complete overwrite of the product quantity prices
 
+    
 * **Arguments:**
-    * *`array`* **changes**:
+    * *`array`* **changes**: 
 * **Examples:**
     * Overwrite the product prices with a new quantity price graduation
 
+        {% raw %}
         ```twig
-        {% do debug.dump(product.calculatedPrice.unit, 'discount') %}
-		
-		{% do product.calculatedPrice.surcharge(10) %}
-		
-		{% do debug.dump(product.calculatedPrice.unit, 'surcharge') %}
+        {% do product.calculatedPrices.change([
+		    { to: 20, price: services.price.create({ 'default': { 'gross': 15, 'net': 15} }) },
+		    { to: 30, price: services.price.create({ 'default': { 'gross': 10, 'net': 10} }) },
+		    { to: null, price: services.price.create({ 'default': { 'gross': 5, 'net': 5} }) },
+		]) %}
         ```
-
+        {% endraw %}
 ### count()
 
 * The `count()` function returns the number of prices which are stored inside this collection.
 
+    
 * **Returns** `int`
 
     Returns the number of prices which are stored inside this collection
-
 _________
-
 ## [`Shopware\Core\Content\Product\Hook\Pricing\ProductProxy`](https://github.com/shopware/shopware/blob/trunk/src/Core/Content/Product/Hook/Pricing/ProductProxy.php) {#productproxy}
 
 The `ProductProxy` is a wrapper for the `SalesChannelProductEntity`. It provides access to all properties of the product,
 but also wraps some data into helper facade classes like `PriceFacade` or `PriceCollectionFacade`.
 
+
 ### __get()
 
-* The `__get()` function allows access to all properties of the [SalesChannelProductEntity](https://github.com/shopware/shopware/blob/trunk/src/Core/Content/Product/SalesChannel/SalesChannelProductEntity.php)
+* The `__get()` function allows access to all properties of the [SalesChannelProductEntity](https://github.com/shopware/platform/blob/trunk/src/Core/Content/Product/SalesChannel/SalesChannelProductEntity.php)
```

Comment:
@mstegmeyer, is it old reference you are using here, maybe? The URL should be shopware/shopware (will end up in a redirect anyways).

Reply:
True, this was wrongly linked in platform (where this is autogenerated).